iib/workers/tasks/build_merge_index_image.py

```diff
@@ -279,25 +280,31 @@ def handle_merge_request(
                 request_id, arch, include_transport=False
             )
 
-            if is_image_fbc(intermediate_image_name):
-                err_msg = 'File-Based catalog image type is not supported for deprecation yet.'
-                log.error(err_msg)
-                raise IIBError(err_msg)
-
-            # opm can only deprecate a bundle image on an existing index image. Build and
-            # push a temporary index image to satisfy this requirement. Any arch will do.
-            # NOTE: we cannot use local builds because opm commands fails,
-            # index image has to be pushed to registry
-            _build_image(temp_dir, 'index.Dockerfile', request_id, arch)
-            _push_image(request_id, arch)
-
-            deprecate_bundles(
-                deprecation_bundles,
-                temp_dir,
-                prebuild_info['binary_image'],
-                intermediate_image_name,
-                overwrite_target_index_token,
-            )
+            # we can check if source index is FBC or not because intermediate_image
+            # will be always the same type because it is built
```

Comment:
Can you elaborate what you mean by same type?

Reply:
by type I meant  SQLite or FBC

Reply:
should I rephrase it ?

Reply:
I meant is type will be one exact thing or same as Source
like if source is X (fbc/sqlite) intermediate will be X
I think it got changed after our discussion, so was confirming that